Program.cs

```diff
@@ -1,10 +1,45 @@
-﻿namespace Training
-{
-    internal class Program
-    {
-        static void Main(string[] args)
-        {
-            Console.WriteLine("Hello, World!");
-        }
-    }
+﻿// ---------------------------------------------------------------------------------------
+// Training ~ A training program for new joinees at Metamation, Batch- July 2023.
+// Copyright (c) Metamation India.                                                
+// ---------------------------------------------------------------------------
+// Program.cs                                                                     
+// Program to print a chess board using unicode values
+// ---------------------------------------------------------------------------------------
+using System.Text;
+using static System.Console;
+
+namespace Training {
+   #region class Program ------------------------------------------------------------------------
+   /// <summary>Chess Board</summary>
+   internal class Program {
+      #region Method ----------------------------------------------
+      /// <summary>This method prints the chess board with all pieces</summary>
+      static void Main () {
+         OutputEncoding = Encoding.UTF8;
+         string blRook = "\u265C", blKnight = "\u265E", blBishop = "\u265D", vLine = "\u2502",
+                whRook = "\u2656", whKnight = "\u2658", whBishop = "\u2657", hLine = "\u2500";
+         // Black : 265F - pawn, 265C - rook, 265E - knight, 265D - bishop, 265B - queen, 265A - king
+         // White : 2659 - pawn, 2656 - rook, 2658 - knight, 2657 - bishop, 2655 - queen, 2654 - king
+
+         for (int i = 0; i < 10; i++) {
+            switch (i) {
+               case 0: WriteLine ("\u250C" + string.Concat (Enumerable.Repeat ($"{hLine}{hLine}{hLine}\u252C", 7)) + $"{hLine}{hLine}{hLine}\u2510"); continue;

```

Comment:
Many cases have this similar line beign repeated. Refactor and move to a method and call it. You could pass unicode char as parameter.